packages/flutter_driver/lib/src/driver.dart

```diff
@@ -118,8 +118,9 @@ class FlutterDriver {
       if (signal == 'timeout') {
         throw new DriverError(
           'Timed out waiting for Flutter Driver extension to become available. '
-          'To enable the driver extension call registerFlutterDriverExtension '
-          'first thing in the main method of your application.'
+          'Ensure your application (lib/main.dart) imports '
```

Comment:
More precisely: `'Ensure your test app imports '`

A test app is not necessarily `lib/main.dart`